spec/status.yml

```diff
@@ -96,7 +96,7 @@ components:
       get:
         tags:
         - 'status'
-        operationId: "authenticatorServiceStatus"
+        operationId: "authenticatorService"
         summary: "Details whether an authentication service has been configured properly"
         description: "Once the status webservice has been properly configured and the relevant user
```

Comment:
We should enumerate in the description the authenticators that support this route. They include:
- Azure Authenticator
- OIDC Authenticator

The following authenticators are NOT supported by this route at this time:
- AWS IAM Authenticator
- Kubernetes Authenticator
- LDAP Authenticator

We can also link to the [documentation](https://docs.conjur.org/Latest/en/Content/Integrations/Authn-status.htm) on configuring the Conjur server to enable certain users and groups to access this endpoint.